test/integration/ethereum/exchangeIssuanceLeveragedIntegration.spec.ts

```diff
@@ -78,6 +78,7 @@ if (process.env.INTEGRATIONTEST) {
           addresses.dexes.uniV2.router,
           addresses.dexes.sushiswap.router,
           addresses.dexes.uniV3.router,
+          addresses.dexes.uniV3.quoter,
```

Comment:
I didn't see a Curve test case anywhere. If you want to add it, it would probably easiest to be done here since here. (In the unittests you'd have to spin up the curve setup)